cooperative-delivery-wg/README.md

```diff
@@ -1,3 +1,12 @@
 # Cooperative Delivery WG 
 
-Please check the [charter draft (WIP)](https://docs.google.com/document/d/1n8AtfQQGJIJrsU-F-oGN6DVXE76E8OVCXsPpXkWgmBo/edit#) for the purpose & goals of the Cooperative Delivery working group. Feel free to add yourself as interested parties and get involved in this effort.
+Please check the [Charter](./charter.md) for the purpose & goals of the Cooperative Delivery working group. Feel free to add yourself as interested parties and get involved in this effort.
+
+### Meetings
+TBD
```

Comment:
Shall we decide a meeting schedule before merging this PR?

Reply:
I think we should decide this in one of the first meetings ...